src/python/WMCore/Services/RucioConMon/RucioConMon.py

```diff
@@ -100,7 +100,7 @@ def getRSEStats(self):
         update timestamps for all RSEs known to CMS Rucio
         :return: A dictionary
         """
-        uri = "/WM/stats"
+        uri = "/stats"
```

Comment:
Todor, can you please check if this leading slash sign `/` can cause problems when using this module?
I have the impression that we would actually be adding 2 slash signs here, one enforced from the super class:
https://github.com/dmwm/WMCore/blob/4f353b944d7f0a8ee7b1a1cc7c2e318827180af4/src/python/WMCore/Services/Service.py#L121

So, my suggestion is to define the API endpoint without is, e.g.:
```
        uri = "stats"
```

The same comment applies to the other 2 lines.